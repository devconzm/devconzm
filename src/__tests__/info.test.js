import React from "react";
import { render } from "react-testing-library";
import InfoPage from '../components/Info'

describe("InfoPage Component", () => {
  const { getByLabelText} = render(<InfoPage />);
  const info = getByLabelText("page-info");

  it("should contain some text", () => {
    expect(info).toHaveTextContent("some lots of text here");
  });
  it('should match the snapshot', () => {
      expect(render(<InfoPage />)).toMatchSnapshot()
  });
});

