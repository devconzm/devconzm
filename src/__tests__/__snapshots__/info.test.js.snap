// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPage Component should match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <p
        aria-label="page-info"
        class="text-center"
      >
        some lots of text here
      </p>
    </div>
  </body>,
  "container": <div>
    <p
      aria-label="page-info"
      class="text-center"
    >
      some lots of text here
    </p>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
